test/utils/FetchMockHelper.js

```diff
@@ -1,5 +1,17 @@
 import fetchMock from 'fetch-mock';
 
+const timeout = (prom, time, ret) =>
```

Comment:
This is a failsafe to make sure that the FetchMockHelper doesn't time out itself.